src/components/intro/intro.scss

```diff
@@ -1,5 +1,12 @@
 @import "../../colors";
 
+h1 {
+    line-height: 2.125rem;
+    color: $ui-orange;
+    font-size: 1.625rem;
+    font-weight: 400;
+}
+
```

Comment:
This should be within the `.intro` block so it won't affect `h1`'s outside of the component.